apps/changelog/management/commands/changelog.py

```diff
@@ -0,0 +1,92 @@
+# Future Imports
+from __future__ import print_function
+
+# Django Imports
+from django.core.management.base import BaseCommand
+
+
+# Django 1 and >2 compatible import
+try:
+    # Django Imports
+    from django.core.urlresolvers import reverse
+except ImportError:
+    from django.urls import reverse
```

Comment:
# Reviewer Note
`reverse` function is being exported in different paths in Django `1` and `>2` versions. This is an import workaround to make it work in both versions.